sesman/chansrv/pulse/Makefile

```diff
@@ -2,8 +2,11 @@
 # build xrdp pulseaudio modules
 #
 
+# change this to your pulseaudio source directory
 PULSE_DIR = /home/lk/pulseaudio-1.1
-CFLAGS    = -Wall -O2 -I$(PULSE_DIR) -I$(PULSE_DIR)/src -DHAVE_CONFIG_H -fPIC
+# change this if you're using non-default socket directory
+SOCK_DIR  = /tmp/.xrdp
+CFLAGS    = -Wall -O2 -I$(PULSE_DIR) -I$(PULSE_DIR)/src -DHAVE_CONFIG_H -fPIC -DXRDP_SOCKET_DIR=\"$(SOCK_DIR)\"
```

Comment:
I guess XRDP_SOCKET_DIR should be XRDP_SOCKET_PATH. Otherwise it does not compile.